.jscsrc

```diff
@@ -2,11 +2,20 @@
   "esnext": true,
   "validateQuoteMarks": "'",
   "requireDotNotation": true,
-  "requireCurlyBraces": true,
   "requireCommaBeforeLineBreak": true,
   "requireOperatorBeforeLineBreak": true,
   "disallowTrailingWhitespace": true,
   "disallowMultipleLineStrings": true,
   "disallowMixedSpacesAndTabs": true,
-  "disallowEmptyBlocks": true
-}
+  "disallowEmptyBlocks": true,
+
```

Comment:
Sure. I'm not sure which is closest to my own coding style, though. I know I dislike airbnb's, and I worked briefly w. Google's and that seemed fine. Any thoughtz?